cmake/kokkos_arch.cmake

```diff
@@ -663,9 +663,19 @@ IF (KOKKOS_ENABLE_OPENMPTARGET)
 ENDIF()
 
 IF (KOKKOS_ENABLE_OPENACC)
-  COMPILER_SPECIFIC_FLAGS(
-    NVHPC -acc
-  )
+  IF(KOKKOS_CUDA_ARCH_FLAG)
+    STRING(REPLACE "sm_" "cc" NVHPC_CUDA_ARCH ${KOKKOS_CUDA_ARCH_FLAG})
+    COMPILER_SPECIFIC_FLAGS(
+      NVHPC -acc -gpu=${NVHPC_CUDA_ARCH}
+    )
+  ELSEIF(KOKKOS_AMDGPU_ARCH_FLAG)
+    # FIXME_OPENACC
+    COMPILER_SPECIFIC_FLAGS(
+      NVHPC -acc
```

Comment:
NVHPC compiler supports AMD CPUs but NOT AMD GPUs.

Reply:
Duh :)
Should I just error out if no CUDA arch was set ?